packages/rum-core/src/performance-monitoring/transaction-service.js

```diff
@@ -94,22 +95,24 @@ class TransactionService {
 
   createPerfOptions(options) {
```

Comment:
nit: probably change it to createOptions? `createPerfOptions` doesn't seem right. Nothing related to the PR, just saying since you are already doing the refactoring.